application/server/connect/connectApiExpress.js

```diff
@@ -118,7 +127,27 @@ module.exports = function(app){
 
   app.post(`${ApiEndpoints.MEMBERS}/:member_guid/identify`, async (req, res) => {
     const ret = await req.connectService.updateConnection(
-      { id: req.params.member_guid, job_type: 'identify' },
+      { id: req.params.member_guid, job_type: 'aggregate_identity' },
+      req.context.resolved_user_id
+    )
+    res.send({
+      members: ret
+    })
+  })
+
+  app.post(`${ApiEndpoints.MEMBERS}/:member_guid/verify`, async (req, res) => {
+    const ret = await req.connectService.updateConnection(
```

Comment:
What will happen if this request fails?

Reply:
The response will include an "error_message" if it fails. If the member status changes to rejected or failed for whatever reason the widget will notify the user.